index.dd

```diff
@@ -336,9 +337,9 @@ digitalmars.com/pnews/indexing.php?server=news.digitalmars.com&group=digitalmars
 [browse]) $(LINK2 news://news.digitalmars.com/digitalmars.D, [usenet])
 $(HTTP lists.puremagic.com/cgi-bin/mailman/listinfo/digitalmars-d,
 [mailing list])) is the best place to discuss anything and everything
-related to D: language design ideas, suggestions, status and future,
-contributions to the language and its standard library. Reach and
-engage all major D contributors, including the very creator of D,
+related to D: language design ideas, suggestions, current and future status,
```

Comment:
"future status" sounds weird, doesn't it? How about "current status and future direction"?